haiku-apps/melt/melt-1.3.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A CD burning application"
+DESCRIPTION="Melt, a CD burning application."
+APP="Melt"
+HOMEPAGE="https://github.com/matthiasr/melt"
+COPYRIGHT="2000 Lukas Hartmann / Atomatrix"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="a32d2888d6bb601e8158b1e0a373e9f52171681b"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="8f420c8a83ae175f618e29c390db83d7bd56eda4c1d788a76f8897b4f120e134"
+SOURCE_DIR="melt-$srcGitRev"
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
```

Comment:
```diff
-SOURCE_DIR="melt-$srcGitRev"
 SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="melt-$srcGitRev"
 
-ARCHITECTURES="x86_gcc2 !x86_64"
+ARCHITECTURES="x86_gcc2 x86_64"
```
* `SOURCE_DIR` should come after `SOURCE_FILENAME`, see the [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).
* Yes, the frontend builds and runs on x86_64 too :)
(I did not test the app. I have only built it on each architecture, installed it and launched it.)

Reply:
Nice! I couldn't test it on x86_64 but had tested it on x86, which worked.